### ISSUE TITLE
Opening/Closing brake violation of psr2

### ISSUE THREAD

**Issue by dereuromark** [slevomat/coding-standard#457]
Your own code/standard violates psr2 in terms of opening and closes class bodies:
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
and the examples state that the content must go after the line of opening and the closing brace on the line after the content.

As such the code seems to be invalid here in terms of whitespace:
https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Arrays/TrailingArrayCommaSniff.php#L13
https://github.com/slevomat/coding-standard/blob/master/SlevomatCodingStandard/Sniffs/Arrays/TrailingArrayCommaSniff.php#L66
```php
class TrailingArrayCommaSniff implements Sniff
{

	public ... {
        }

}
```
should be
```php
class TrailingArrayCommaSniff implements Sniff
{
	public ... {
        }
}
```
